### PR TITLE
intx: Add as_bytes() helper for casting intx to bytes

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

File: test/unittests/test_intx.cpp [hunk 1]
```diff
@@ -490,4 +490,18 @@ TYPED_TEST(uint_test, to_string_base)
...
-}
\ No newline at end of file
+}
+
+TYPED_TEST(uint_test, as_bytes)
+{
+    constexpr auto x = TypeParam{0xa05};
+    const auto b = as_bytes(x);
+    EXPECT_EQ(b[0], 5);
+    EXPECT_EQ(b[1], 0xa);
```
Comment (axic) [inline test/unittests/test_intx.cpp]: This codifies it as little endian. Should this be implemented for both be/le ?

Comment (chfast) [reply test/unittests/test_intx.cpp]: I have to add big endian support in general. But not a priority in the moment. I only have time to add features needed in other EVM projects.

Comment (axic) [reply test/unittests/test_intx.cpp]: Well EVM is big endian, isn't it?

Comment (chfast) [reply test/unittests/test_intx.cpp]: Correct, but this API is needed for `intx::be` API and in one place in evmone. It exposes internal representation as bytes.

Comment (axic) [reply test/unittests/test_intx.cpp]: So perhaps can you move `as_bytes` into `intx::be` ?

**Pull request merged**